x-pack/plugins/security_solution/public/explore/components/risk_score/enable_risk_score/index.tsx

```diff
@@ -64,23 +63,12 @@ const EnableRiskScoreComponent = ({
         }
         actions={
           <EuiToolTip content={!signalIndexExists ? i18n.ENABLE_RISK_SCORE_POPOVER : null}>
-            {isDeprecated ? (
-              <RiskScoreUpgradeButton
```

Comment:
I think this is the only usage of the button. If that is true you could also delete `kibana/x-pack/plugins/security_solution/public/explore/components/risk_score/risk_score_onboarding/risk_score_upgrade_button.tsx`